code/lib/cli/src/automigrate/index.ts

```diff
@@ -280,12 +295,26 @@ export async function runFixes({
               },
             }
           );
+        } else if (promptType === 'notified') {
```

Comment:
```suggestion
        } else if (promptType === 'notification') {
```

Reply:
Also needs updating in a few other places

Reply:
VSCode's symbol renaming seems not worked out :(